NavViewsAndOptionSelector/KPIndex.h

```diff
@@ -0,0 +1,22 @@
+//
+//  KPIndex.h
+//  NavViewsAndOptionSelector
+//
+//  Created by MacMan on 8/14/15.
+//  Copyright (c) 2015 MacManApp. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface KPIndex : UIViewController
```

Comment:
Why does `KPIndex` inherit from `UIViewController`? It doesn't seem to act as a view controller, but rather as a simple object with a couple of properties. I'd suggest making it inherit from `NSObject` instead.

Also, I'm a bit confused by the naming choice. An `index` is most frequently used to refer to an integer denoting the position of an element in a sequence or other data structure, not the element itself. If you want a generic name for an element in a list, something like `ListItem` or `Element` would be more descriptive.